src/MOIWrapper.jl

```diff
@@ -136,30 +170,38 @@ function update_bounds_for_binary_vars!(col_lb::Vector{Float64}, col_ub::Vector{
     end
 end
 
-function update_zeroone_indices(user_optimizer::MOI.ModelLike, mapping::MOIU.IndexMap,
-    ci::Vector{MOI.ConstraintIndex{F,S}}, zero_one_indices::Vector{Int}) where {F, S}
+function update_zeroone_indices(user_optimizer::MOI.ModelLike, 
+                                mapping::MOIU.IndexMap, 
+                                ci::Vector{MOI.ConstraintIndex{F,S}}, 
+                                zero_one_indices::Vector{Int}) where {F, S}
+                                
     for i in 1:length(ci)
         f = MOI.get(user_optimizer, MOI.ConstraintFunction(), ci[i])
         push!(zero_one_indices, mapping.varmap[f.variable].value)
     end
 end
 
-function update_integer_indices(user_optimizer::MOI.ModelLike, mapping::MOIU.IndexMap,
-    ci::Vector{MOI.ConstraintIndex{F,S}}, integer_indices::Vector{Int}) where {F, S}
+function update_integer_indices(user_optimizer::MOI.ModelLike, 
+                                mapping::MOIU.IndexMap,
+                                ci::Vector{MOI.ConstraintIndex{F,S}}, 
+                                integer_indices::Vector{Int}) where {F, S}
+                                
     for i in 1:length(ci)
         f = MOI.get(user_optimizer, MOI.ConstraintFunction(), ci[i])
         push!(integer_indices, mapping.varmap[f.variable].value)
     end
 end
 
 """
-    function copy!(cbc_optimizer, user_optimizer; copynames=false)
+    function copy_to(cbc_optimizer, user_optimizer; copy_names=false)
 
-Receive a cbc_optimizer which contains the pointer to the cbc C object and instantiate the object cbc_model_format::CbcModelFormat based on user_optimizer::AbstractModel (also provided by the user).
-Function loadProblem of CbcCInterface requires all information stored in cbc_model_format.
+Receive a cbc_optimizer which contains the pointer to the cbc C object and 
+instantiate the object cbc_model_format::CbcModelFormat based on 
```

Comment:
This documentation is a bit confusing. Aren't these implementation details that callers of `copy_to` don't need to know?

Reply:
you are right, i removed it